i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md

```diff
@@ -2,46 +2,506 @@
+| --ssh-backend           | Define the backend you want to use. It can be: sshcli (default), plink and libssh.                                                                                     |
+| --ssh-username          | Define the user name to log in to the host.                                                                                                                            |
+| --ssh-password          | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use --ssh-priv-key instead.   |
+| --ssh-port              | Define the TCP port on which SSH is listening.                                                                                                                         |
+| --ssh-priv-key          | Define the private key file to use for user authentication.                                                                                                            |
+| --sshcli-command        | ssh command (default: 'ssh').                                                                                                                                          |
+| --sshcli-path           | ssh command path (default: none)                                                                                                                                       |
+| --sshcli-option         | Specify ssh cli options (example: --sshcli-option='-o=StrictHostKeyChecking=no').                                                                                      |
+| --plink-command         | plink command (default: 'plink').                                                                                                                                      |
+| --plink-path            | plink command path (default: none)                                                                                                                                     |
+| --plink-option          | Specify plink options (example: --plink-option='-T').                                                                                                                  |
+| --libssh-strict-connect | Connection won't be OK even if there is a problem (server known changed or server found other) with the ssh server.                                                    |
+| --filter-name           | Filter app name (can be a regexp).                                                                                                                                     |
+| --filter-counters       | Only display some counters (regexp can be used). Example: --filter-counters='calls'                                                                                    |
+| --warning-*-count       | /persecondWarning threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                            |
```

Comment:
```suggestion
| --warning-*-count       | /persecond Warning threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                            |
```

Reply:
I think there's a parsing issue here. we should remove /persecond